pages/docs/tutorials/kafka/index.md

```diff
@@ -0,0 +1,188 @@
+  eastKafkaServer:
+    host: east.mykafkacluster.org:8092
+    description: Kafka broker running in east warehouse
+    protocol: kafka
+```
+
+In the above snippet:
+
+- The `asyncapi` field indicates that you are using AsyncAPI version 3.0.0.
+  
+- The `info` field provides information about the API. Here the the APIs `title`, `version`, and `description` are being defined.
+  
+- The `server` field specifies the details of the server, including the `host`, `description`, and the `protocol` that is being used i.e., Kafka. This section allows you to define multiple clusters or servers, as shown in the provided code block.
+
+<Remember>
+If an application has multiple servers hidden from the user behind a load balancer, then only the address of the load balancer needs to be included in the AsyncAPI document. 
+</Remember>
+
+### Define channels and operations
+
+Next, let's move on to the `channels` and `operations` section. The channel addresses are the topics in Kafka, they are the routes to which your API will be sending/receiving messages. The `operations` property is used to describe how your application interacts with the channels.
+
+```
+operations:
+  onUserSignedUp:
+    action: receive
+    channel:
+      $ref: '#/channels/userSignedUp'
+
+
+channels:
+  userSignedUp:
+    description: This channel contains a message per each user who signs up in our application.
+    address: user_signedup
+    messages:
+      userSignedUp:
+        $ref: '#/components/messages/userSignedUp'
+```
+
+In the above snippet:
+
+- The `onUserSignedUp` object specifies the nature of the operation. The `action` property indicates that our application will be receiving the information. The `channel` property points to the channel where the operation occurs.
+  
+- The `userSignedUp` object inside `channels` describes the Kafka topic where our application will be receiving the information and the associated message definition. The `address` field represents the actual name of the Kafka topic. The `messages` field describes the expected messages in that topic. 
+
+### Define messages and schemas
+
+Finally, you'll define the messages and their payload. The payload defines how the event would look like when it's sent from the channel.
+
+```
+components:
+  messages:
+    userSignedUp:
+      payload:
+        type: object
+        properties:
+          user-id:
+            type: integer
+            description: This property describes the id of the user
+          user-email:
+            type: string
+            description: This property describes the id of the user
+```
+
+In the above snippet:
+
+- The `userSignedUp` message is defined which describes the payload (content) of the message.
+  
+- The `payload` property defines the content of the message using [JSON Schema](https://json-schema.org/). It means that your message payload should contain a `user-id` which is an integer and a `user-email` property which is a string property.
```

Comment:
you use AsyncAPI schema, not JSON schema here, AsyncAPI schema is default

if you would use JSON schema, you would have to do it like in case of Avro schema, with `schemaFormat`